crates/sui-indexer/src/processors/transaction_stats_processor.rs

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use prometheus::Registry;
+use std::sync::Arc;
+use tracing::info;
+
+use sui_indexer::errors::IndexerError;
+use sui_indexer::metrics::IndexerTransactionStatsProcessorMetrics;
+use sui_indexer::models::transaction_stats::{commit_transaction_stats, NewTransactionStats};
+use sui_indexer::models::transactions::{read_last_transaction, read_transactions_since};
+use sui_indexer::{get_pg_pool_connection, PgConnectionPool};
+
+const TRANSACTION_STATS_COMPUTATION_WINDOW_IN_SECONDS: i64 = 5;
```

Comment:
Why is this a fixed number?

Reply:
explorer wants TPS to be calculated in a sliding time window of roughly 5 seconds